api/registerOrganisation/index.ts

```diff
@@ -38,6 +38,26 @@ export function mapRequestObject(requestBody: RegistrationData): RegistrationReq
       value: requestBody.otherServices
     });
   }
+  if (requestBody.regulators && requestBody.regulators.length > 0) {
+    let regIndex = 0;
+    requestBody.regulators.map((regulator) => {
+      request.orgAttributes.push({
+        key: `regulators-${regIndex}`,
+        value: JSON.stringify(regulator)
+      });
+      regIndex++;
+    });
+  }
+  if (requestBody.individualRegulators && requestBody.individualRegulators.length > 0) {
+    let iRegIndex = 0;
```

Comment:
map function provides a function by default. You dnt need to add an index for it
`const numbers = [1, 2, 3, 4];
const filteredNumbers = numbers.map((num, index) => {
  if (index < 3) {
    return num;
  }
});`